thundra/integrations/mongo.py

```diff
@@ -0,0 +1,101 @@
+import traceback
+import json
+import time
+import logging
+from thundra import config, constants
+from thundra.plugins.invocation import invocation_support
+from thundra.integrations.base_integration import BaseIntegration
+from pymongo.monitoring import CommandListener
+from thundra.opentracing.tracer import ThundraTracer
+from bson.json_util import dumps
+
+logger = logging.getLogger(__name__)
+
+
+class CommandTracer(CommandListener):
+    _scopes = {}
+
+    def started(self, event):
+        tracer = ThundraTracer.get_instance()
+        if not tracer.get_active_span():
+            return
+
+        scope = tracer.start_active_span(operation_name=event.database_name, finish_on_close=False)
+
+        self._scopes[event.request_id] = scope
+        span = scope.span
+
+        # Inject before span tags
+        try:
+            command_name = event.command_name
+            scope.span.class_name = constants.ClassNames['MONGO']
+            scope.span.domain_name = constants.DomainNames['DB']
+
+            try:
+                host, port = event.connection_id
+            except:
+                host, port = "", ""
+
+            operation_type = constants.MongoDBCommandTypes.get(command_name.upper(), '')
+
+            tags = {
+                constants.SpanTags['OPERATION_TYPE']: operation_type,
+                constants.DBTags['DB_TYPE']: 'mongo',
+                constants.DBTags['DB_HOST']: host,
+                constants.DBTags['DB_PORT']: port,
+                constants.SpanTags['DB_INSTANCE']: event.database_name,
+                constants.MongoDBTags['MONGO_COMMAND_NAME']: command_name,
```

Comment:
Can we also get collection name information. Collections are tables like in a traditional database systems